### PR TITLE
Added solution to number of integers between 0 and n containing the digit d

### LINKED ISSUE #9: Find the Number which contains the digit d
Take a user input n and d and prints numbers between 0 to n which contains specific digit d.

### PR DESCRIPTION
**Pull request by KoushikSahu**
**Fixes #9**

Complexity: O(n*number_of_digits_in_n)

### CODE REVIEW

**Comment by varchasa** [comment]
Hi @KoushikSahu, your code is perfect just do one change - print all the numbers also.

**Comment by KoushikSahu** [comment]
@varchasa I have done the changes. Please review again and let me know if there is anything to modify or rectify.

File: has_digit_d.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    print("Enter value of d")
+    d = int(input())
+
+    ans = 0
+
+    print("The integers between 0 and n having digit d are:")
+    for i in range(0, n+1):
+        if(has_digit_d(i, d)==True):
+            print(i)
```
Comment (KoushikSahu) [inline has_digit_d.py:22]: @varchasa line 22 would print out the desired number

**Comment by KoushikSahu** [comment]
@varchasa Please add a "hacktoberfest-accepted" label to this PR so that it would count as legitimate PR for hacktoberfest. Thank you in advance!

**Review by varchasa — APPROVED** [review]

**Pull request closed without merging**